src/AirshipBindings.iOS.MessageCenter/ApiDefinitions.cs

```diff
@@ -240,6 +240,7 @@ interface UAMessageCenter
         // + (null_unspecified instancetype)shared;
         [Static]
         [Export("shared")]
+        [New]
```

Comment:
what's the deal with these `[New]` annotations?

Reply:
`[New]` clears up a warning in the build. Basically, the [warning](https://stackoverflow.com/questions/19193821/use-the-new-keyword-if-hiding-was-intended-warning) is "Shared() hides inherited member Shared(). Use the new keyword if hiding was intended.". The problem is that we need the subclass `Shared()` methods, as they have the correct return type. If we don't have them, then the calling C# has to cast UAComponent to the correct return type.

Reply:
got it, thanks